detox/src/devices/common/drivers/android/tools/ApkValidator.js

```diff
@@ -0,0 +1,52 @@
+const DetoxRuntimeError = require('../../../../../errors/DetoxRuntimeError');
+
+const setupGuideHint = 'For further assistance, visit the Android setup guide: https://github.com/wix/Detox/blob/master/docs/Introduction.Android.md';
```

Comment:
I really think that we should change all of our links to the website. However, you can keep it as you want for now and we can change it in a separated PR with all other links. I opened an issue (https://github.com/wix/Detox/issues/3117) for that, but it should be an easy change.

Note that there are already several links to the web so the experience you described is already broken :)

Reply:
If you think otherwise let's discuss it on this in the issue, I don't think it should block us with this PR.

Reply:
Let's do it in the scope of #3117 (great that it's there, btw)